### DISCUSSION TITLE
Improve List Filters in Pods::filters()

### DISCUSSION THREAD

**Discussion by sc0ttkclark** [pods-framework/pods#6603] [Ideas]
more customizable & versatile on searching / filtering (ie: by date - month/year, etc..)
## Field coverage
- [x] Relationship dropdown fields (pick)
- [ ] Associated Taxonomies (filter by taxonomy terms) for Post Types (taxonomy)
- [ ] Date / Time ranges (date / datetime / time)
- [ ] Number ranges (number / currency)
- [ ] Boolean checkbox (boolean)
- [ ] Text fields (code, color, email, paragraph, phone, slug, text, website, wysiwyg)
- [ ] Option to use a checkbox for checking for a non-empty value or not (all fields, specifically useful to check if file / avatar fields are set)

**Comment by sc0ttkclark** [comment]
Pushed to 2.x

**Comment by aurel-appsthru** [comment]
I make the same wish ...
And Imho, it would be interesting to choose the columns/fields to display/filter in list views.

**Comment by sc0ttkclark** [comment]
@nicdford said he's brave, let's put him to the test :)

**Comment by nicdford** [comment]
My head was bigger than my sword.
